### PR TITLE
Issue#73: Fixed examples in the Readme

### PR DESCRIPTION
**Pull request by harshm-dev**
- Examples with hadoop as the source to use register option since migrate isn't supported
- Arctic Nessie prod URL pattern

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -216,7 +216,7 @@ which can be used for actual migration using the `--identifiers-from-file` optio
 
 Sample input:
 ```shell
-java -jar iceberg-catalog-migrator-cli-0.2.0.jar migrate \
+java -jar iceberg-catalog-migrator-cli-0.2.0.jar register \
```
Comment (ajantha-bhat) [inline README.md:219]: The below text descriptions has to be changed from `migaration` to `registration`, `migrate` to `register`, `migrated` to `registered`
These examples were captured before blocking migration on Hadoop. 

or easier way is to use HIVE as source catalog for these scenario examples. 

I can do it in follow up If you don't have time for this.

Comment (harshm-dev) [reply README.md]: Thanks, can you please review again now.

File: README.md [hunk 2]
```diff
@@ -299,43 +299,43 @@ This can help users understand why the migration failed.
...
-If necessary, users can manually remove these tables from the source catalog or attempt a bulk migration to transfer all tables from the source catalog.
+To recover, users can manually remove these tables from the source catalog or attempt a bulk migration to transfer all tables from the source catalog.
 
 ### B.5) Users need to move away from one catalog to another with selective tables (maybe want to move only the production tables, test tables, etc)
 
 Users can provide the selective list of identifiers to migrate using any of these 3 options
 `--identifiers`, `--identifiers-from-file`, `--identifier-regex` and it can be used along with the dry-run option too.
 
-Sample input: (only migrate tables that starts with "foo.")
+Sample input: (only register tables that starts with "foo.")
```
Comment (ajantha-bhat) [inline README.md:313]: migrate

**Review by ajantha-bhat — APPROVED** [review]
LGTM. Thanks for fixing this.

**Pull request merged**